src/lamp_py/ingestion/compress_gtfs/pq_to_sqlite.py

```diff
@@ -0,0 +1,82 @@
+import os
+import sqlite3
+
+import pyarrow
+import pyarrow.dataset as pd
+
+from lamp_py.runtime_utils.process_logger import ProcessLogger
+from lamp_py.ingestion.utils import gzip_file
+
+
+def sqlite_type(pq_type: str) -> str:
+    """
+    return SQLITE type from pyarrow Field type
+    """
+    if "int" in pq_type:
+        return "INTEGER"
+    if "bool" in pq_type:
+        return "INTEGER"
+    if "float" in pq_type:
+        return "REAL"
+    if "double" in pq_type:
+        return "REAL"
+    return "TEXT"
+
+
+def sqlite_table_query(table_name: str, schema: pyarrow.Schema) -> str:
+    """
+    return CREATE TABLE query for sqlite table from pyarrow schema
+    """
+    logger = ProcessLogger("sqlite_create_table")
```

Comment:
seems like we're logging how long it takes to generate the query here rather than how long it takes to create and populate the table which might be more useful.

Reply:
Dropped the logging of the full `CREATE TABLE` query. 

Should be able to track the duration of SQLITE table creation/insertion when `pq_folder_to_sqlite`  calls `logger.add_metadata(current_file=file)`